README.md

```diff
@@ -59,14 +59,29 @@ Add in your `config_prod.yml` file, you don't need error notifier when you are i
 ```yml
 # app/config/config_prod.yml
 elao_error_notifier:
-    from: from@example.com
-    to: to@example.com
+    from: from@example.com # required if using "default_mailer" notifier
+                           # ignored if `notifiers.mailer.from` is set
+    to: to@example.com # required if using "default_mailer" notifier
+                       # can be a string or an array or email addresses
+                       # ignored if `notifiers.mailer.to` is set
```

Comment:
Shouldn't we ditch the elao_error_notifier.to and elao_error_notifier.from ? and set the default notifier to the mailer?